src/smc-webapp/editor_chat.cjsx

```diff
@@ -1022,7 +1022,7 @@ ChatRoom = (name) -> rclass
                 <Row>
                     <Col xs={10} style={padding:'0px 2px 0px 2px'}>
                         <Input
-                            autoFocus   = {true}
+                            autoFocus   = {if isMobile.Android() then true else false}
```

Comment:
oh, and more general, maybe you both actually want it the other way around ? I'm trying to test it.